env/build_test.go

```diff
@@ -31,14 +39,28 @@ func testBuildEnv(t *testing.T, when spec.G, it spec.S) {
 			})
 			out := benv.List()
 			sort.Strings(out)
-			if s := cmp.Diff(out, []string{
+			expectedVars := []string{
 				"CNB_STACK_ID=some-stack-id",
 				"CPATH=some-cpath",
+				"HOME=some-home",
+				"HOSTNAME=some-hostname",
+				"HTTPS_PROXY=some-https-proxy",
+				"HTTP_PROXY=some-http-proxy",
 				"LD_LIBRARY_PATH=some-ld-library-path",
 				"LIBRARY_PATH=some-library-path",
+				"NO_PROXY=some-no-proxy",
 				"PATH=some-path",
 				"PKG_CONFIG_PATH=some-pkg-config-path",
-			}); s != "" {
+			}
+			if runtime.GOOS != "windows" {
+				expectedVars = append(
```

Comment:
It took me a while to figure out why/how this expectation works. Case-sensitivity right? Might be worth calling out in a comment.

Reply:
Sounds good!